ui/app/components/distribution-bar.js

```diff
@@ -66,14 +66,18 @@ export default Component.extend(WindowResizable, {
     this.renderChart();
   },
 
+  updateChart: observer('_data.@each.{value,label,className}', function() {
+    this.renderChart();
```

Comment:
So with the more live updating - does `this.renderChart()` need to be an e-c task? Just curious if there's a small chance of race between updating the data more frequently, drawing the chart, and navigating away from the page/component.

Reply:
The chart rendering is entirely synchronous, so it should be fine.